CouchbaseLite/Document.py

```diff
@@ -78,7 +78,9 @@ def getProperties(self):
 
     @property
     def JSON(self):
-        return sliceToString(lib.CBLDocument_PropertiesAsJSON(self._ref))
+        # TODO: c-api call doesn't exist any more, CBLDocument_PropertiesAsJSON
+        #       need to test the following to see if it works
+        return sliceToString(lib.CBLDocument_Properties(self._ref))
```

Comment:
That will not work. CBLDocument_Properties returns a FLDict pointer, not a slice. You'll need to call FLValue_ToJSON on it to create a JSON string. That FLSliceResult needs to be released afterwards.

Reply:
I took a different approach and simply calling encodeJson instead. Also, I did run the python tests and they ran without error.